[2021-12-18 09:13:16,624] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.git_add 2021-12-18T09:03:37.133567+00:00 [queued]>
[2021-12-18 09:13:16,634] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.git_add 2021-12-18T09:03:37.133567+00:00 [queued]>
[2021-12-18 09:13:16,639] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-18 09:13:16,640] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-18 09:13:16,640] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-18 09:13:16,664] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): git_add> on 2021-12-18T09:03:37.133567+00:00
[2021-12-18 09:13:16,667] {standard_task_runner.py:52} INFO - Started process 85953 to run task
[2021-12-18 09:13:16,679] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'rank_update', 'git_add', '2021-12-18T09:03:37.133567+00:00', '--job-id', '62', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/rank_update.py', '--cfg-path', '/tmp/tmpdtff6jno', '--error-file', '/tmp/tmpbyl79pzr']
[2021-12-18 09:13:16,681] {standard_task_runner.py:77} INFO - Job 62: Subtask git_add
[2021-12-18 09:13:16,771] {logging_mixin.py:104} INFO - Running <TaskInstance: rank_update.git_add 2021-12-18T09:03:37.133567+00:00 [running]> on host airflowvm
[2021-12-18 09:13:16,972] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=rank_update
AIRFLOW_CTX_TASK_ID=git_add
AIRFLOW_CTX_EXECUTION_DATE=2021-12-18T09:03:37.133567+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-18T09:03:37.133567+00:00
[2021-12-18 09:13:16,972] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-18 09:13:16,973] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'git -C /home/airflow/Test/ add rank.md']
[2021-12-18 09:13:16,988] {subprocess.py:75} INFO - Output:
[2021-12-18 09:13:16,993] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-18 09:13:17,023] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=rank_update, task_id=git_add, execution_date=20211218T090337, start_date=20211218T091316, end_date=20211218T091317
[2021-12-18 09:13:17,073] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-18 09:13:17,092] {local_task_job.py:151} INFO - Task exited with return code 0
